Dockerfile.windows

```diff
@@ -250,18 +256,89 @@ RUN `
   Remove-Item C:\gitsetup.zip; `
   `
   Write-Host INFO: Creating source directory...; `
-  New-Item -ItemType Directory -Path ${GOPATH}\src\github.com\docker\docker | Out-Null; `
+  #TODO - Per https://docs.docker.com/engine/reference/builder/ 
+  #       the below line is redundant and should be removed.
+  #New-Item -ItemType Directory -Path ${GOPATH}\src\github.com\docker\docker | Out-Null; `
   `
   Write-Host INFO: Configuring git core.autocrlf...; `
-  C:\git\cmd\git config --global core.autocrlf true; `
+  C:\git\cmd\git config --global core.autocrlf true;
+
+# Ensure all directories exist that we will require below....
+WORKDIR ${GOPATH}\${DOCKER_SRC_REL_LOCATION}\${BUNDLES_REL_LOCATION}
```

Comment:
See my comment above, I don't think we should abuse `WORKDIR`'s  side-effect as a replacement for `mkdir`

Reply:
Removed as it was there for its side-effect alone